get-started/installation-procedure.md

```diff
@@ -56,6 +56,32 @@ On most **Linux** distributions before the installation can be started, the user
     ![KDE setting executible](../images/installation/exec-kde.jpg)
 
 
+## Create an Account
+
+Fiddler Everywhere account enables you to save and share sessions with collaborators and additionally provides access to the Pro features (if Pro version is purchased). Signing in is mandatory for both Free and Pro versions. You can sign up by [creating an unique Fiddler account](#create-new-fiddler-account) or by [using existing Google account](#use-google-account).
```

Comment:
1) **The** Fiddler Everywhere account
2) This paragraph may need to be restructured. The most important part is that having an account is mandatory to use Fiddler and you should consider making that the first sentence. Also, are we planning on listing pro features in some of the following articles? If not, mentioning them here is not very helpful.

Reply:
One of the upcoming articles will be about FREE vs PRO version comparison .. I am waiting currently working on this one  and once ready I will create slug and link the new article where needed (including here)